cmd/semdiffstat/main.go

```diff
@@ -49,22 +50,23 @@ func main() {
 		log.Fatalf("could not parse Go files %v and %v: %v\n", aName, bName, err)
 	}
 
+	w := new(tabwriter.Writer)
+	w.Init(os.Stdout, 0, 8, 0, '\t', 0)
 	for _, c := range changes {
-		// TODO: align | characters
 		// TODO: general UI improvements
 		if c.Inserted {
-			fmt.Printf("%v (inserted) %s\n", c.Name, pluses(c.InsLines))
+			fmt.Fprintf(w, "%v\t | %s (inserted)\n", c.Name, pluses(c.InsLines))
```

Comment:
I think you want a trailing `\t` at the end of each printed line.

Reply:
What do you mean? The output breaks if I add a trailing \t, looks fine otherwise.

Reply:
Oh yes, I see it in the docs...